segmentstore/server/src/test/java/io/pravega/segmentstore/server/containers/StreamSegmentContainerTests.java

```diff
@@ -1881,11 +1881,17 @@ private void checkActiveSegments(SegmentContainer container, int expectedCount)
         val initialActiveSegments = container.getActiveSegments();
         int ignoredSegments = 0;
         for (SegmentProperties sp : initialActiveSegments) {
-            if (sp.getName().equals(EXPECTED_METADATA_SEGMENT_NAME)) {
+            boolean match = false;
+            for (String systemSegment : SystemJournal.getChunkStorageSystemSegments(container.getId())) {
```

Comment:
Does this include the one you just removed from here? That's the container metadata segment which has nothing to do with chunks.

Reply:
initialActiveSegments includes EXPECTED_METADATA_SEGMENT_NAME.

which is NameUtils.getMetadataSegmentName(CONTAINER_ID);